Frameworks/UIKit/StarboardXaml/ApplicationMain.mm

```diff
@@ -147,7 +147,7 @@ int ApplicationMainStart(const char* principalName,
     [displayMode _updateDisplaySettings];
 
     UIApplicationMainInit(principalClassName, delegateClassName, defaultOrientation, (int)activationType, activationArgument);
-    ScheduleMainRunLoop();
```

Comment:
>ScheduleMainRunLoop [](start = 4, length = 19)

This was taking > 20s for internal test app.  With async, we take that cost after app activation so PLM doesn't kill us.